### ISSUE TITLE
node process eats CPU even when no site activity is happening

### ISSUE THREAD

**Issue by jspeis** [DataUSA/datausa-site#811]
Was looking at `saguaro` and noticed CPU load was slightly high on the server. The top process appears to be the canon node process?

```
deploy@saguaro:~$ top -b -n1 -c  | awk '/PID *USER/{print;getline;print}'
   PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
366934 deploy    20   0 1627496 762948  23896 R 100.0  1.2   8484:27 node /home/deploy/datausa-site/node_modules/.bin/canon-start
```

I checked the HTTP access logs and did not see requests had been made within the last ~15 minutes, but the process seems to continues to appear to eat CPU at a high level. I'm wondering if some of this may be related to uncaught errors that are spinning async handlers?

Looks like there's something called `node-tick` that might help debug? https://stackoverflow.com/questions/10167477/how-to-debug-node-js-causing-100-cpu-usage

Also I was not able see any logs from the process in the journalctl

**Comment by jspeis** [comment]
cc @davelandry @jhmullen still noticing this on zion. it seems like it might be some like of process leak happening inside the site somewhere? because shortly after process restart and with light usage it doesn't seem to happen but after the site's process has been running for a while we'll see the `canon-start` process pegged to 100% on a CPU

<img width="877" alt="Screen Shot 2019-03-29 at 9 15 59 AM" src="https://user-images.githubusercontent.com/87581/55235219-6981bb00-5203-11e9-9fe8-2c71c23b6c29.png">

**Comment by davelandry** [comment]
@jspeis I went in and modified the runtime script for canon on `zion` to include the internal Node profiler: `node --prof index.js`

Now, a log file is being updated so that we can parse to see what's happening:
1. run `node --prof-process isolate-0x4235130-v8.log > static/processed.txt`
2. view `static/processed.txt` (available on the web at `https://zion.datausa.io/processed.txt`)

It contains a "Summary" section by language which is good place to start. Here's a blog post I found that explains how to read parts of it: https://nodejs.org/en/docs/guides/simple-profiling/
